.github/workflows/supply-chain-security-validation.yaml

```diff
@@ -114,6 +114,10 @@ jobs:
         with:
           config-file: ${{ inputs.codeql-code-scanning-config-file }}
           languages: ${{ fromJSON(steps.languages.outputs.result) }}
+      - name: Add coopnorge Nuget source
+        if: steps.languages.outputs.result == 'csharp'
```

Comment:
`steps.languages.outputs.result` can contain multiple values separated bu `,`. You probably want to use [`contains`](https://docs.github.com/en/actions/learn-github-actions/expressions#contains)